iothub/service/src/ServiceClient.cs

```diff
@@ -99,111 +100,74 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
         }
 
         /// <summary>
-        /// Open the ServiceClient instance
+        /// Open the ServiceClient instance.
         /// </summary>
-        /// <returns></returns>
         public abstract Task OpenAsync();
 
         /// <summary>
-        /// Close the ServiceClient instance
+        /// Close the ServiceClient instance.
         /// </summary>
-        /// <returns></returns>
         public abstract Task CloseAsync();
 
         /// <summary>
-        /// Send a one-way notification to the specified device
+        /// Send a cloud to device message to the specified device.
         /// </summary>
-        /// <param name="deviceId">The device identifier for the target device</param>
-        /// <param name="message">The message containing the notification</param>
-        /// <param name="timeout">The operation timeout override. If not used uses OperationTimeout default</param>
-        /// <returns></returns>
+        /// <param name="deviceId">The device identifier for the target device.</param>
+        /// <param name="message">The cloud to device message.</param>
+        /// <param name="timeout">The operation timeout which defaults to 1 minute, if unspecified.</param>
         public abstract Task SendAsync(string deviceId, Message message, TimeSpan? timeout = null);
 
         /// <summary>
-        /// Removes all messages from a device's queue.
+        /// Removes all cloud to device messages from a device's queue.
         /// </summary>
-        /// <param name="deviceId"></param>
-        /// <returns></returns>
-        public abstract Task<PurgeMessageQueueResult> PurgeMessageQueueAsync(string deviceId);
+        /// <param name="deviceId">The device identifier for the target device.</param>
+        /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
+        public abstract Task<PurgeMessageQueueResult> PurgeMessageQueueAsync(string deviceId, CancellationToken cancellationToken = default);
```

Comment:
CancellationToken cancellationToken = default is the same as CancellationToken.None